### ISSUE TITLE
UnboundLocalError 

### ISSUE THREAD

**Issue by shunn337** [ranaroussi/yfinance#923]
I frequently get errors like the one below. It's happened both with and without multithreading. The stack trace is slightly different when threads=True/False, but multi.py line 202 & base.py line 208 are common. I'm running yfinance 0.1.68 on OSX using python 3.9.4 installed from Homebrew.  

```
Traceback (most recent call last):
  File "/usr/local/Cellar/python@3.9/3.9.4/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/local/Cellar/python@3.9/3.9.4/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 892, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.9/site-packages/multitasking/__init__.py", line 104, in _run_via_pool
    return callee(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/yfinance/multi.py", line 188, in _download_one_threaded
    data = _download_one(ticker, start, end, auto_adjust, back_adjust,
  File "/usr/local/lib/python3.9/site-packages/yfinance/multi.py", line 202, in _download_one
    return Ticker(ticker).history(period=period, interval=interval,
  File "/usr/local/lib/python3.9/site-packages/yfinance/base.py", line 208, in history
    if "chart" in data and data["chart"]["error"]:
UnboundLocalError: local variable 'data' referenced before assignment 
```

**Comment by george-kuanli-peng** [comment]
I also have this problem.

**Comment by LemonCANDY42** [comment]
I have this problem too.

**Comment by forgottenecho** [comment]
I have this problem when trying to use a proxy for the history() function

**Comment by maltin1234** [comment]
Same

**Comment by ValueRaider** [comment]
Appears to be fixed in version 0.1.69 (#920)
